### PR TITLE
add Map of Indonesia

### PR DESCRIPTION
**Pull request by tyangjawi03**
Hello Victor,
I'd add map of Indonesia.

### CODE REVIEW

**Comment by VictorCazanave** [comment]
Thanks a lot for your contribution @tyangjawi03 ! 😃 

I'll try to review it next week.

File: packages/indonesia/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Credits
+
+This map is based on the work of [MapSVG](https://mapsvg.com).
+
+The original map is available [here](https://mapsvg.com/maps/indonesia) under the [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/) license.
+
+### Changes
+
+* Remove unnecessary attributes
```
Comment (VictorCazanave) [inline packages/indonesia/README.md:59]: Could you list the changes you really did please?

File: packages/indonesia/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# 🇮🇩 Indonesia
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains all the states and union territories of Indonesia:
```
Comment (VictorCazanave) [inline packages/indonesia/README.md:5]: I think they are called "provinces" (source: https://en.wikipedia.org/wiki/Provinces_of_Indonesia)

File: packages/indonesia/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* Aceh
```
Comment (VictorCazanave) [inline packages/indonesia/README.md:6]: Could you use the English names please? (in the SVG too)

**Review by VictorCazanave — COMMENTED** [review]
Some things to fix, then it's good to be merged

**Comment by tyangjawi03** [comment]
Hi Victor,

already refactoring code to follow your advice.

thanks

**Comment by VictorCazanave** [comment]
Thanks again!

@all-contributors please add @tyangjawi03 for code

**Pull request merged**